src/data/mv_polynomial.lean

```diff
@@ -1635,4 +1822,17 @@ by rw [pderivative_mul, pderivative_C, zero_mul, zero_add]
 
 end pderivative
 
+section counit
+
+variables [comm_semiring α] [comm_semiring β] [algebra α β]
+variables (α β)
+
+noncomputable def counit : mv_polynomial β α →ₐ β :=
```

Comment:
Docstring? And maybe even a module docstring if this is a stub of a new section on coalgebra stuff. 

In fact, this file is a bit long, so whenever it gets broken up, this would probably end up in a new file.

Reply:
Oops, I didn't even mean to include this.